src/fluree/crypto.cljc

```
@@ -234,50 +234,53 @@
+    With account ID as key identifier:
+      (create-jws \"payload\" \"162259eb44ebceca49e00bcc95496a2eeba5528886414859c95a3ee045cbd1f5\" {:account-id true})
     
     With custom key ID:
       (create-jws \"payload\" private-key-hex {:kid \"my-key-1\"})"
-  ([payload signing-key] (create-jws payload signing-key {}))
+  ([payload signing-key]
+   (create-jws payload signing-key {}))
   ([payload signing-key opts]
    (jws/serialize-jws payload signing-key opts)))
 
 (defn ^:export verify-jws
   "Verify a JWS (JSON Web Signature) using Ed25519. Can provide public key or extract from JWS header.
   
-  Returns result map if valid, or Exception object if verification fails.
+  Returns result map if valid, throws exception if verification fails.
   
   Parameters:
   - jws: JWS string in compact format
   - public-key (optional): Ed25519 public key. If nil, will try to extract from JWS header
   
   Returns on success: {:payload payload :pubkey pubkey :header header-map :kid kid-if-present}
-  Returns on error: Exception with details
+  Throws on error: Exception with details
   
   Examples:
     With provided public key:
       (let [result (verify-jws jws-string pubkey)]
-        (if (instance? Exception result)
-          (handle-error result)
-          (use-payload (:payload result))))
+        (use-payload (:payload result)))
     
     Extract public key from JWS header (requires kid or jwk in header):
       (let [result (verify-jws jws-string)]
-        (if (instance? Exception result)
-          (handle-error result)
-          (do (println \"Verified with key:\" (:kid result))
-              (use-payload (:payload result)))))"
+        (println \"Verified with key:\" (:kid result))
+        (use-payload (:payload result)))"
   ([jws] (verify-jws jws nil))
   ([jws public-key]
-   (jws/verify jws public-key)))
\ No newline at end of file
+   (let [result (jws/verify jws public-key)]
+     (if (instance? #?(:clj Exception :cljs js/Error) result)
```

Instead of catching and returning an error value in `jws/verify`, can we just throw from there? This seems redundant.